cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -113,6 +115,7 @@ func (p *Provisioner) Provision(opts pvController.VolumeOptions) (*v1.Persistent
 
 	//TODO: Determine if hostpath or device based Local PV should be created
 	stgType := pvCASConfig.GetStorageType()
+	sendEventOrIgnore(name, stgType, analytics.LocalPVProvision)
```

Comment:
How does a developer determine if this is the right place to invoke this function `sendEventOrIgnore`

Reply:
Why can't it be inside ProvisionHostPath?

Reply:
> How does a developer determine if this is the right place ...

This seems like a good place, we'd have pv*(local-pv properties)* details(viz storage-type, size, pv-name) which would be sent to Google Analytics.

> Why can't it be inside ProvisionHostPath?

Because we need to send events for both `device` and `hostpath` storage-types, we can add it there(`ProvisionHostPath`) along-with `ProvisionBlockDevice`